.gitignore

```diff
@@ -3,3 +3,5 @@
 Cargo.lock
 index.node
 node_modules
+.DS_Store
```

Comment:
what is `.DS_Store` ?

Reply:
It's a file that is created by the mac indexer.